README.md

```diff
@@ -693,6 +693,17 @@ Specifies the number of seconds an IP address is in cooldown after pod deletion.
 **Note:** 0 is a supported value, however it is highly discouraged.
 **Note:** Higher cooldown periods may lead to a higher number of EC2 API calls as IPs are in cooldown cache.
 
+----
+
+#### `DISABLE_CONTAINER_V6` (v1.15.0+)
+
+Type: Boolean as a String
+
+Default: `false`
+
+When `DISABLE_CONTAINER_V6` is set, the [tuning plugin](https://www.cni.dev/plugins/current/meta/tuning/) is chained and configured to disable IPv6 networking in each newly created container network namespace. Set this variable when you have an IPv4 cluster and containerized applications that cannot tolerate IPv6 being enabled.
+Container runtimes such as `containerd` will enable IPv6 in newly created container network namespaces regardless of host settings.
```

Comment:
Let's add a blurb around Multus as well (i.e.,) user needs to make sure that any chained plugins are not dependent on this setting and/or not modifying these settings.